Resources/Prototypes/Entities/Constructible/Specific/Conveyor/constructionGraphs.yml

```diff
@@ -0,0 +1,57 @@
+﻿- type: constructionGraph
+  id: ConveyorGraph
+  start: start
+  graph:
+    - node: start
+      edges:
+        - to: entity
+          steps:
+            - component: ConveyorItem
+              icon:
+                sprite: Constructible/Power/conveyor.rsi
+                state: conveyor_loose
+              name: conveyor belt assembly
+              doAfter: 2
+    - node: item
+      entity: ConveyorBeltItem
+      actions:
+        - !type:SetAnchor
+          value: false
+    - node: entity
+      entity: ConveyorBelt
+      actions:
+        - !type:SetAnchor
+          value: true
+        - !type:SnapToGrid
+          offset: Center
+      edges:
+        - to: item
+          steps:
+            - tool: Prying
+              doAfter: 3
+
+- type: constructionGraph
+  id: leverDefinitelyNotCopiedFromGirderNoNoNo
```

Comment:
Maybe some better name?

Reply:
zumo insisted on the name. sorry, above my paygrade...